pages/basic-features/social-login.mdx

```diff
@@ -0,0 +1,80 @@
+The functionality is powered by the [Laravel Socialite](https://laravel.com/docs/socialite) package and its extensive range of providers.
+
+## Getting Started
+
+To enable social login in your application, follow these simple steps:
+
+<div className="steps-container">
+### Select which services you want to use and enable them in `config/craftable-pro.php`:
+```php filename="config/craftable-pro.php"
+   'social_login' => [
+        'allowed_services' => [
+            'microsoft' => false,
+            'github' => true,
+            'google' => true,
+            'twitter' => false,
+            'facebook' => false,
+            'apple' => false,
+        ],
+        'self_registration' => [
+            // define if users can self register into craftable pro interface
+            'enabled' => true,
+            // and if enabled, then which role(s) they should have assigned by default. Use role names here.
+            // It can be a string for one role or an array for multiple roles.
+            'default_role' => 'Administrator',
+            'allowed_domains' => ['craftable.pro'], // use * for allowing any domain
+        ]
+    ]
+```
+### If you enable social login self-registration, don't forget to also check the `allowed_domains`.
+### Register event listeners for your enabled providers in `app/Providers/EventServiceProvider.php`:
+```text filename="app/Providers/EventServiceProvider.php"
+protected $listen = [
+    \SocialiteProviders\Manager\SocialiteWasCalled::class => [
+        \SocialiteProviders\Apple\MicrosoftExtendSocialite::class,
+        \SocialiteProviders\Apple\GitHubExtendSocialite::class,
+        \SocialiteProviders\Apple\TwitterExtendSocialite::class,
+        \SocialiteProviders\Apple\FacebookExtendSocialite::class,
+        \SocialiteProviders\Apple\GoogleExtendSocialite::class,
+        \SocialiteProviders\Apple\AppleExtendSocialite::class,
+    ]
+];
+```
+### Get client credentials from the providers and put them in your `.env`:
+```php filename=".env"
+MICROSOFT_CLIENT_ID=
+MICROSOFT_CLIENT_SECRET=
+MICROSOFT_REDIRECT_URI="/admin/login/microsoft/callback"
+
+GITHUB_CLIENT_ID=
+GITHUB_CLIENT_SECRET=
+GITHUB_REDIRECT_URI="/admin/login/github/callback"
+
+GOOGLE_CLIENT_ID=
+GOOGLE_CLIENT_SECRET=
+GOOGLE_REDIRECT_URL="/admin/login/google/callback"
+
+TWITTER_CLIENT_ID=
+TWITTER_CLIENT_SECRET=
+TWITTER_REDIRECT_URI="/admin/login/twitter/callback"
+
+FACEBOOK_CLIENT_ID=
+FACEBOOK_CLIENT_SECRET=
+FACEBOOK_REDIRECT_URI="/admin/login/facebook/callback"
+
+APPLE_CLIENT_ID=
+APPLE_CLIENT_SECRET=
+APPLE_REDIRECT_URI="/admin/login/apple/callback"
+```
+</div>
+
+## Other providers
+
+To use another existing provider, check out the [Laravel Socialite](https://laravel.com/docs/socialite) or [Laravel Socialite Providers](https://socialiteproviders.com/) documentation.
```

Comment:
I would also mention what the user should do in the craftable pro config. Based on this: https://github.com/BRACKETS-by-TRIAD/craftable-pro/pull/344/files#diff-66081ea1503cc9859c7b27727b9741d861e51bee20a5301884bf1e0e85dd7e2b I would say that adding the new service to allowed services should be enough. Plus implementation of the social login icon to the login page.